assets/css/style.css

```diff
@@ -396,3 +396,108 @@ html[dir='rtl'] .pull-right:not(#rtlChecker) { /* sass-lint:disable-line class-n
 html[dir='rtl'] .pull-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
     float: right !important; /* sass-lint:disable-line no-important */
 }
+.posRelative {
```

Comment:
These rules should be organized into the correct sections in this stylesheet as per the comments, e.g. none of this belongs under the `/* RTL Stuff */` section.